.github/workflows/pull_request.yaml

```diff
@@ -15,4 +15,6 @@ jobs:
       - uses: DeterminateSystems/magic-nix-cache-action@main
         with:
           upstream-cache: https://ngi.cachix.org/
-      - run: nix flake check
+      - run: nix flake check --impure
```

Comment:
Don't do impure if all you want is some unfree packages. Please use `config.allowUnfreePredicate` (via `flake.nix`). Introduce a set of unfree packages, and for each unfree package, leave a comment justifying it.

Reply:
Consider resolving. The package with unfree deps is not present.